src/chrome/content/rules/5ch.xml

```diff
@@ -0,0 +1,45 @@
+<ruleset name="5ch.net">
+        <target host="5ch.net" />
+        <target host="*.5ch.net" />
+
+        <test url="http://5ch.net/" />
+        <test url="http://www2.5ch.net/" />
+        <test url="http://itest.5ch.net/" />
+        <test url="http://premium.5ch.net/" />
+        <test url="http://be.5ch.net/" />
+        <test url="http://i.5ch.net/" />
+        <test url="http://headline.5ch.net/" />
+        <test url="http://newsnavi.5ch.net" />
+        <test url="http://info.5ch.net" />
+        <test url="http://o.5ch.net/" />
+        <test url="http://stat.5ch.net" />
+        <test url="http://dig.5ch.net/" />
+        <test url="http://qb5.5ch.net/" />
+        
+        <test url="http://agree.5ch.net/" />
+        <test url="http://asahi.5ch.net/" />
+        <test url="http://egg.5ch.net/" />
+        <test url="http://fate.5ch.net/" />
+        <test url="http://hayabusa9.5ch.net/" />
+        <test url="http://hebi.5ch.net/" />
+        <test url="http://himawari.5ch.net/" />
+        <test url="http://krsw.5ch.net/" />
+        <test url="http://lavender.5ch.net/" />
+        <test url="http://leia.5ch.net/" />
+        <test url="http://mao.5ch.net/" />
+        <test url="http://matsuri.5ch.net/" />
+        <test url="http://medaka.5ch.net/" />
+        <test url="http://mevius.5ch.net/" />
+        <test url="http://nhk2.5ch.net/" />
+        <test url="http://rio2016.5ch.net/" />
+        <test url="http://rosie.5ch.net/" />
+        <test url="http://swallow.5ch.net/" />
+        <test url="http://tanuki.5ch.net/" />
+
+        <securecookie host="^\." name="^(?:__cfduid|cf_clearance)$" />
+        <securecookie host="^\w" name=".+" />
```

Comment:
You may use the below `securecookie` to protect all of the cookies set by `5ch.net`. Since a wildcard is used, this should be fine and offer better protection. 

```suggestion
        <securecookie host=".+" name=".+" />
```